other-docs/guides/updating-php/README.md

```diff
@@ -11,11 +11,15 @@ There are 2 key steps to getting ready for a new version of PHP:
 
 |Altis|PHP Version|
 |-|-|
-|v7+|7.4|
+|v13+|8.0|
+|v11-v13|7.4-8.0|
```

Comment:
I don't read this in the same way as its written further down. 

it says v11, 12 and 13 have support for php 7.4 and 8.0. At the same time versions v14  and above ( v13+) have php8 only. 

That is not the same as the text written in the other file.

Reply:
Since v11 is not full php8 support, can we use @rmccue's suggestion of something like this: 
- v11 : 7.4- 8.0 (experimental ) 
- v12 : 7.4 - 8.0
- v13 : 8.0

Reply:
Ah yeah I messed that up, think we want to say v13 requires PHP 8 on reflection